sebastian-rueda/tarea-1/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Time Magazine, November 2016</title>
+</head>
+<body>
+    <main class="cover">
+        <figure class="container">
```

Comment:
@SebastianRuedaGlobant , please research about the correct use of figure and figcaption